src/pages/en/reference/directives-reference.md

```diff
@@ -67,6 +67,25 @@ const cmsContent = await fetchHTMLFromMyCMS();
 <Fragment set:html={cmsContent}>
 ```
 
+`set:html={Promise<string>}` injects an HTML string into an element that is wrapped in a promise.
```

Comment:
For terms like Response, Request, and others, we generally capitalize the word, so I think it's nice to follow the same pattern here. It also makes it easier for people from other languages to recognize we are not saying "promise" but rather the JavaScript object "Promise"
```suggestion
`set:html={Promise<string>}` injects an HTML string into an element that is wrapped in a Promise.
```